docs/platform-setup.md

```diff
@@ -18,3 +18,126 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Configure Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Configuring docker registry with nexus should follow this steps:
```

Comment:
I think you can remove this sentence, it is clear that we need to follow each step.

Reply:
changes applied